CmdStarter.Tests.Commands/GlobalOptions/Filters/GlobalOptionFilterExcludeAll.cs

```diff
@@ -0,0 +1,21 @@
+﻿using com.cyberinternauts.csharp.CmdStarter.Tests.Common.Interfaces;
+
+namespace com.cyberinternauts.csharp.CmdStarter.Tests.Commands.GlobalOptions
```

Comment:
Under `Tests.Commands.Options` you used `Attributes` and here `Filters`. I prefer your new name as `Attributes` used one namespace above is structured with sub folders with the name of the attribute, but not under `Options.Attributes`. What do you think?

P.S. In fact, I prefer all the way the names used in `GlobalOptions.Filters` than the ones in `Options.Attributes`. The idea is to be consistent. What do you suggest?

Reply:
Should I rename the `Options.Attributes` to be like `GlobalOptions.Filters`?

Reply:
That's what I preconize. Do you agree?